### PR TITLE
fix(QuickNotes-Menu):Added Create A Task

### PR DESCRIPTION
**Pull request by sanjaykrishna1203**
### Functionality:

- Added Create A Task from Quick Notes
- Changed the colour of the quick notes button
- Made the recent changes to get saved if the user clicks creates a task before saving the changes.
- Added "No Notes Available if No Notes are available.
- Added "Create A Note" from the edit menu.

### Solution:

- Used PopUpHandlerService to Create A Task
- Used EventEmitter for toggling the AddNewTask

### Risk level:
- [x] high 
- [ ] medium
- [ ] low

### How to test:

- Check if "No Notes Available" is shown when no notes are added.
- Add a note and click create a task and check whether the title and description are shown in the respective input fields of the Create new task modal.
- Do the same for the EditNote
- In the edit note component click on the + icon and check whether it takes to the add new note Component.
- Click create a task without saving the changes and check whether the changes are saved.

### CODE REVIEW

**Review by viv9k — APPROVED** [review]
Approved

File: src/app/body/quick-notes/addnew-note/addnew-note.component.ts [hunk 1]
```diff
@@ -25,17 +26,26 @@ import { ToolsService } from 'src/app/services/tool/tools.service';
 export class AddnewNoteComponent implements OnInit {
 
   @Output() addNoteCompleted = new EventEmitter<boolean>();
+  @Output() showList = new EventEmitter();
```
Comment (Pendyala-thanvi) [inline src/app/body/quick-notes/addnew-note/addnew-note.component.ts:29]: In which parent component are we using this event emitter.

File: src/app/body/quick-notes/edit-note/edit-note.component.ts [hunk 2]
```diff
@@ -29,18 +30,28 @@ import { ErrorHandlerService } from 'src/app/services/error-handler/error-handle
 export class EditNoteComponent implements OnInit {
 
   @Output() editNoteCompleted = new EventEmitter<boolean>();
+  @Output() addNewNote = new EventEmitter();
```
Comment (Pendyala-thanvi) [inline src/app/body/quick-notes/edit-note/edit-note.component.ts:33]: Same here

File: src/app/body/quick-notes/addnew-note/addnew-note.component.ts (hunk 1 above)
Comment (sanjaykrishna1203) [reply src/app/body/quick-notes/addnew-note/addnew-note.component.ts]: Line Removed.

**Review by Pendyala-thanvi — APPROVED** [review]
Approved

**Pull request merged**